pkg/util/log/clog.go

```diff
@@ -930,6 +931,16 @@ func (sb *syncBuffer) rotateFile(now time.Time) error {
 		return err
 	}
 
+	// Redirect stderr to the current INFO log file in order to capture panic
```

Comment:
Using INFO instead of ERROR doesn't seem right, although I guess we can't duplicate the stack trace across multiple files like a real ERROR log would. (Is it even worth preserving the google-style multi-file logging output? I don't think our log levels are currently coherent enough for ERROR or WARNING files to be useful on their own, especially if stack traces are going to be INFO-only) 

Should we put stderr in a fourth file? (That's what borg did)